venter/migrations/0007_auto_20181123_1240.py

```diff
@@ -16,4 +16,8 @@ class Migration(migrations.Migration):
             name='authority_email',
             field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='venter.Authorities'),
         ),
+        migrations.RemoveField(
+            model_name='complaints',
+            name='authority_email',
```

Comment:
**You cannot edit migrations!**. Should be fixed simply by reverting all migrations to master and `makemigrations` again.

Reply:
Fixed

Reply:
Great 👍